lib/grpnames.gd

```diff
@@ -24,8 +24,8 @@
 ##
 ##  <Description>
 ##    For normal subgroups <A>U</A> and <A>V</A> of <A>G</A>,
-##    IsTrivialNormalIntersection returns
-##    true if <A>U</A> and <A>V</A> intersect trivially, and false otherwise.
+##    <Ref Oper="IsTrivialNormalIntersection"/> returns <K>true</K> if
```

Comment:
This "self-referencing" is a valid question - I've also asked it at some point. It happens that GAPDoc renders references in a clever way, understanding when they refer to the same mansection or to another one. Compare, for example, 
```
##  <ManSection>
##  <Var Name="GaussianIntegers"/>
##
##  <Description>
##  <Ref Var="GaussianIntegers"/> is the ring <M>&ZZ;[\sqrt{{-1}}]</M>
##  of Gaussian integers.
##  This is a subring of the cyclotomic field
##  <Ref Func="GaussianRationals"/>.
##  </Description>
##  </ManSection>
```
with its documentation [here](http://www.gap-system.org/Manuals/doc/ref/chap60.html#X80BD5EAB879F096E).

The rationale behind this is (as I see it - aybe @frankluebeck had other considerations in mind while developing GAPDoc) as follows. Using just `IsTrivialNormalIntersection` is incorrect: one should use GAPDoc markup at least to distinguish it by using different font with `<C>IsTrivialNormalIntersection</C>`. But:
-  it may happen that the text will be moved to another place in the manual later, or copied and pasted as a template for another GAPDoc section. Then one should take care and replace `<C>` by `<Ref>` to get a proper cross-reference. Using `<Ref>` from the very start allows to avoid such hassles in the future. 
- one could of course argue that moving `<C>IsTrivialNormalIntersection</C>` to another location will not cause any problems, because it will be rendered successfully anyway. But then there will be no cross-reference to a documented thing - the quality of the documentation will suffer. It's a good practice to put cross-references to everything which is documented. If you try to cross-ref undocumented thing, you will see the warning. When you use just  `<C>` elements, you will never be warned, while it's good to care whether you mention documented or undocumented things.
- finally, this will guarantee the correct behaviour if some future version of GAPDoc (or just rebuilding the manual with another settings) will change the style of these elements so that `<C>` and `<Ref>` will be displayed differently.